jax/numpy/lax_numpy.py

```diff
@@ -1153,6 +1153,17 @@ def ravel(a, order="C"):
   return reshape(a, (size(a),), order)
 
 
+@_wraps(onp.unravel_index)
```

Comment:
Add a bit of documentation about the differences in behavior for negative and out-of-bounds indices (using the `lax_description` argument)